build.gradle

```diff
@@ -86,10 +97,10 @@ publishing {
             version version.toString()
             pom.withXml {
                 def root = asNode()
-                root.appendNode("name", "spiglin")
+                root.appendNode("name:spiglin")
                 root.appendNode("description",
                         "A collection of Kotlin extensions and utilities for the Spigot/Bukkit framework.")
-                root.appendNode("url", "https://github.com/johnnyjayjay/spiglin/")
+                root.appendNode("url:https://github.com/johnnyjayjay/spiglin/")
```

Comment:
Is there a particular reason for these changes?

Reply:
I don't even remember even touching those lines not sure why they're changed

Reply:
Oh I think I copied a dependencyspec that looked like this implementation "group", "artifact", "version" and replaced ", " with :